### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by akbarsazish**
1- Add style to match the original design.

### CODE REVIEW

File: src/components/Navigation.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
+import { Link } from 'react-router-dom';
+import avatar from './assets/avatar.png';
+
+function Navigation() {
+  return (
+    <nav className="nav">
+      <ul className="nav-items">
+        <li className="nav-link">
+          <Link to="/" className="link">
+            <h1 className="logo"> BookStor CMS </h1>
```
Comment (souad988) [inline src/components/Navigation.js]: kindly correct the typo in this title BookStore

File: src/components/Navigation.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+          </Link>
+        </li>
+        <li className="nav-link">
+          <Link to="/" className="link"> Books </Link>
+        </li>
+        <li className="nav-link">
+          <Link to="/categories" className="link"> Categories </Link>
+        </li>
```
Comment (souad988) [inline src/components/Navigation.js]: Your navigation list looks good but please make sure to use the same color and add the active effect to match the template 

Capture d’écran du 2023-08-04 22-17-16
Capture d’écran du 2023-08-04 22-17-07

File: src/components/BookContainer.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  return (
+    <div className="container">
+      {books.map((book) => (
+        <div key={book.item_id}>
+          <Book book={book} />
+        </div>
+      ))}
+      <div className="part-devider"></div>
+      <AddBookButton app_id={app_id} />
+    </div>
```
Comment (souad988) [inline src/components/BookContainer.js:25]: Great work by creating this component 👍🏼  ,  one very minor styling adjustement needed is to reduce the width of this container or increase its margin 

**figma design**
Capture d’écran du 2023-08-04 22-19-52

**your design**
Capture d’écran du 2023-08-04 22-19-27

File: src/components/Book.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+      </div>
+       <div className="book-item"> 
+         <div className="book-improvment">
+            <div className="circle">
+            </div>
+            <div className="percentage">
+              <p className="competion-percentage"> 80 % </p> 
+              <p className="completed"> completed </p>
+            </div>
+          </div>
```
Comment (souad988) [inline src/components/Book.js]: the book component looks great only this element needs to be aligned to the right also inrease its size 

Capture d’écran du 2023-08-04 22-26-53
Capture d’écran du 2023-08-04 22-26-42

File: src/components/BookForm.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+  return (
+    <div className="book-form">
+      <h2 className="form-title">ADD NEW BOOK</h2>
+      <form onSubmit={handleSubmit} className="form">
+        <InputForm type="text" id="inputTitle" placeholder="Book title" isRequired />
+        <InputForm type="text" id="inputAuthor" placeholder="Book Author" isRequired />
+        <button type="submit" id="add-book-btn">ADD BOOK</button>
+      </form>
+    </div>
```
Comment (souad988) [inline src/components/BookForm.js:25]: Great work done by creating the InputForm component 🥇 , I have one very very small styling issues to mention : 

please make sure to use the same color for the title and the input placeholders  do your best to stick to the design 

Capture d’écran du 2023-08-04 22-28-10


Capture d’écran du 2023-08-04 22-27-58

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @akbarsazish  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

242722065-db428b0c-8dfc-49c4-a156-42f227e32868



## Highlights
Create reusable components 👍🏼 
Good PR description


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by akbarsazish** [comment]
Hi @souad988 
Thank you so much for taking the time to review my code. I really appreciate your feedback. Your warm recommedation make the design better.

**Review by franclobo — APPROVED** [review]
Hi @akbarsazish, 

# STATUS: APPROVED 😃 

git-merge

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- No inter errors ✔️ 
- Descriptive PR ✔️ 
- Correct Git flow ✔️ 
- Correct styling ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**